html-forms/index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+</head>
+<body>
+  Email
+  <form action="none">
+    <input type="text" name="WhyCoding" id="WhyIsCodingCool">
```

Comment:
You are missing the `label` for this `input`. This is meant to be for the email.